interfaces/builtin/repowerd.go

```diff
@@ -0,0 +1,307 @@
+  bus=system
+  path=/com/canonical/Unity/Screen
+  peer=(label=unconfined),
+`)
+
+var repowerdPermanentSlotSecComp = []byte(`
+# Description: Allow operating as the repowerd service. Reserved because this
+# gives
+#  privileged access to the system.
+# Usage: reserved
+
+# Can communicate with DBus system service
+sendmsg
+sendto
+recvfrom
+recvmsg
+`)
+
+var repowerdConnectedPlugSecComp = []byte(`
+# Description: Allow using repowerd service. Reserved because this gives
+#  privileged access to the repowerd service.
+# Usage: reserved
+
+# Can communicate with DBus system service
+sendmsg
+sendto
+recvfrom
+recvmsg
+`)
+
+var repowerdPermanentSlotDBus = []byte(`
+<!-- DBus policy for repowerd (upstream version 2016.12) -->
+<policy user="root">
+  <allow own="com.canonical.Unity.Screen"/>
+  <allow send_destination="com.canonical.Unity.Screen"
+         send_interface="com.canonical.Unity.Screen"/>
+
+  <allow own="com.canonical.powerd"/>
+  <allow send_destination="com.canonical.powerd"
+         send_interface="com.canonical.powerd"/>
+</policy>
+
+<policy context="default">
+  <allow send_destination="com.canonical.Unity.Screen"
+         send_interface="org.freedesktop.DBus.Introspectable" />
+  <allow send_destination="com.canonical.Unity.Screen"
+         send_interface="org.freedesktop.DBus.Properties"
+         send_type="method_call" send_member="Get" />
+  <allow send_destination="com.canonical.Unity.Screen"
+         send_interface="org.freedesktop.DBus.Properties"
+         send_type="method_call" send_member="GetAll" />
+
+  <allow send_destination="com.canonical.powerd"
+         send_interface="org.freedesktop.DBus.Introspectable" />
+  <allow send_destination="com.canonical.powerd"
+         send_interface="org.freedesktop.DBus.Properties"
+         send_type="method_call" send_member="Get" />
+  <allow send_destination="com.canonical.powerd"
+         send_interface="org.freedesktop.DBus.Properties"
+         send_type="method_call" send_member="GetAll" />
+
+  <allow send_destination="com.canonical.powerd"
+         send_interface="com.canonical.powerd"
+         send_type="method_call" send_member="userAutobrightnessEnable" />
+  <allow send_destination="com.canonical.powerd"
+         send_interface="com.canonical.powerd"
+         send_type="method_call" send_member="getBrightnessParams" />
+  <allow send_destination="com.canonical.powerd"
+         send_interface="com.canonical.powerd"
+         send_type="method_call" send_member="setUserBrightness" />
+  <allow send_destination="com.canonical.powerd"
+         send_interface="com.canonical.powerd"
+         send_type="method_call" send_member="userAutobrightnessEnable" />
+</policy>
+
+<!-- Allow any user at console (adb shell) to send commands this is
+     needed for unlocking the screen in tests (LP: 1298869) -->
+<policy at_console="true">
+  <allow send_destination="com.canonical.Unity.Screen"/>
+  <allow send_destination="com.canonical.powerd"/>
+</policy>
```

Comment:
I don't think this phablet workaround is what we want here. Touch devices had many assumptions that snapd does not. Please remove this and whenever polkit is implemented, add appropriate polkit policy for devices where this makes sense. cc @vosst